services/kommander/0.2.0/defaults/cm.yaml

```diff
@@ -64,6 +64,8 @@ data:
           repository: ${kommanderLicensingControllerWebhookImageRepository}
       defaultEnterpriseApps:
         centralized-kubecost: "0.22.1"
+        kubecost: "0.22.1"
```

Comment:
Is having `kubecost` ins `defaultApps` few lines below https://github.com/mesosphere/kommander-applications/pull/238/files#diff-50b1ab93c1d40ed8185f0a8ff4d8f44b3b348f75ca3c43d9b8d069d24d9e2782L91 still fine ?

Reply:
I _think_ this should be fine for the mgmt cluster because we set that "default apps deployed" label to true on kommander workspace so the default apps controller ignores deploying those default apps altogether to the mgmt cluster (since we use cli to deploy those, not controller)

Reply:
so the field is nested as
```
attached:
      prerequisites:
        defaultApps
```
so its fine to deploy `kubecost` to attached clusters ?!

Reply:
<discussed offline> this is fine as multi cluster is an enterprise only feature.